apps/browser/qml/pages/components/HistoryContextMenu.qml

```diff
@@ -0,0 +1,45 @@
+/****************************************************************************
+**
+** Copyright (c) 2014 - 2021 Jolla Ltd.
+** Copyright (c) 2021 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.1
+import Sailfish.Silica 1.0
+
+Component {
```

Comment:
What's the reason for wrapping this in a `Component`?

Reply:
Calling the context menu does not work without a wrapper:  ```[W] unknown:92 - file:///usr/lib/qt5/qml/Sailfish/Silica/private/ViewItem.qml:92: TypeError: Cannot call method 'connect' of undefined```

Reply:
Thanks for clarifying. That's peculiar though as I don't get that behaviour here: if I remove the wrapper it works without any errors.

Reply:
Just to clarify, I was only curious, it's not a problem as long as there's a reason for it.

Reply:
If you remove the Component, then nothing will change, but if you use this code directly in `HistoryList.qml`, you will get the error above.

Reply:
But the error is very strange, before everything worked in ` HistoryList.qml`